src/stage1/ir.cpp

```diff
@@ -15500,7 +15500,7 @@ static Stage1AirInst *ir_analyze_struct_field_ptr(IrAnalyze *ira, Scope *scope,
                 is_const, is_volatile, PtrLenSingle, field->align,
                 (uint32_t)(ptr_bit_offset + field->bit_offset_in_host),
                 (uint32_t)host_int_bytes_for_result_type, false);
-        
+
```

Comment:
maybe this is an unnecessary change?

Reply:
what's wrong with removing trailing whitespace? it's clearly a mistake from an editor not configured to remove them on save.

Reply:
nothing wrong, I don't suggest a change, I made just a question, right? 

this can be a mistake fixed, or just a mistake added. I just asked to be sure!

Reply:
Oh yeah, my emacs is setup to remove trailing whitespace, do we want this back in?

Reply:
For me it is fine! I just point out this change!